django_material_demo/cms/polls/views/question.py

```diff
@@ -267,8 +277,40 @@ class QuestionListView(ListModelView, ListFilterView):
     filterset_class = QuestionFilter
 
 
+class QuestionDetailView(DetailModelView):
+    def get_object_data(self):
+        question = super().get_object()
+        thumbnail_name = Question._meta.get_field('thumbnail')
+        thumbnail_name = thumbnail_name.verbose_name.title()
+        for item in super().get_object_data():
+            if item[0] == thumbnail_name:
+                # Skip if no image
+                if item[1]:
+                    #TODO: replace with template
+                    image_html = mark_safe(
+                        f"<img class='thumbnail' src='{item[1].url}' "
+                        f"alt='{item[1].name}'>")
```

Comment:
pls render it in a separate html template file instead of inline html in python.